src/modules/logcollector/include/logcollector.hpp

```diff
@@ -81,6 +82,9 @@ class Logcollector {
     /// @param configurationParser Configuration parser
     void SetupFileReader(const std::shared_ptr<const configuration::ConfigurationParser> configurationParser);
 
+    /// @brief Clean all readers
+    void CleanAllReaders();
```

Comment:
Private?

Reply:
I will keep it this way as it could be useful in future usages.